.changelog

```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- Added SLE15 SP7 as supported client
```

Comment:
Do not abbrev product names ;) We want to cut and paste this to our `.changes` files.
```suggestion
- Added SUSE Linux Enterprise 15 SP7 as supported client
```
You actually added SUSE Linux Enterprise *Server*.  I'm not sure whether this makes a difference.